core-bundle/src/Resources/views/Frontend/preview_toolbar_base.html.twig

```diff
@@ -9,6 +9,9 @@
             <input type="hidden" name="REQUEST_TOKEN" value="{{ request_token }}">
             <div>
                 <button type="button" id="copyPublishedPath" class="tl_submit">{{ 'MSC.copyURL'|trans([], 'contao_default') }}</button>
+                {% if share is not empty %}
+                <a href="{{ share }}" target="_blank" class="tl_submit" onclick="window.open(this.href+'&url='+location.href); return false">{{ 'MSC.shareURL'|trans([], 'contao_default') }}</a>
```

Comment:
```suggestion
                <a href="{{ share }}" target="_blank" class="tl_submit" onclick="window.open(this.href+'&amp;url='+location.href); return false">{{ 'MSC.shareURL'|trans([], 'contao_default') }}</a>
```

Reply:
Are you sure that the ampersand has to be encoded in the `onclick` attribute, too?

Reply:
pretty sure not. This is JS code, not HTML. `this.href` is already decoded when retrieving it from the DOM. But I didn't test it manually to be fair.

Reply:
> Are you sure that the ampersand has to be encoded in the `onclick` attribute, too?

Yes it has to be encoded.